<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">

 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">

  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>

    <width>800</width>
    <height>600</height>

   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="accComboBox">
    <property name="geometry">
     <rect>

      <x>40</x>
      <y>30</y>

      <width>94</width>
      <height>22</height>
     </rect>
    </property>

   </widget>
   <widget class="QPushButton" name="call_account">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌평가잔고내역 확인</string>
    </property>

   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>

      <x>39</x>
      <y>79</y>
      <width>471</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QLabel" name="label_4">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="2" column="2">
      <widget class="QLabel" name="label_3">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="0" column="2">
      <widget class="QLabel" name="label_13">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="0" column="4">
      <widget class="QLabel" name="label_15">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="0" column="1">
      <widget class="QLabel" name="label_12">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="0" column="3">
      <widget class="QLabel" name="label_14">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="0" column="0">
      <widget class="QLabel" name="label_11">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="2" column="1">
      <widget class="QLabel" name="label_2">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="label_1">

       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>

     <item row="2" column="4">

      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>

   <widget class="QTableWidget" name="stocklistTableWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>471</width>
      <height>192</height>

     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="acc_manage">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>93</width>

      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌 관리</string>
    </property>
   </widget>

   <widget class="QTableWidget" name="Danger_wd">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>256</width>
      <height>351</height>

     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>

     <width>800</width>

     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
